server/lib/config/pbench-server-default.cfg

```diff
@@ -2,7 +2,6 @@
 
 ###########################################################################
 # These should be overridden in the env-specific config file.
-default-host = pbench.example.com
```

Comment:
Are you sure you want to remove this line?  Doing so means that, instead of overriding the definition of `default-host` in the env-specific config file, that file now has to override `admin-email` and `mailfrom` specifically and individually (i.e., they won't get set to `${default-user}@${default-host}` by default because there won't be a `default-host` (or `host`) anymore).

Ditto lines 167-175.

Reply:
All this is going away and I just wanted to reduce the steps for updating host names by the container scripts to simplify things.